nisystemlink/clients/file/models/_file_metadata.py

```diff
@@ -0,0 +1,74 @@
+from datetime import datetime
+from typing import Dict, Optional
+
+from nisystemlink.clients.core._uplink._json_model import JsonModel
+from pydantic import Field
+
+from ._link import Link
+
+
+class FileMetadata(JsonModel):
+    field_links: Optional[Dict[str, Link]] = Field(None, alias="_links")
+    """
+    The links to access and manipulate the file:
+    - data: Link to download the file using a GET request
+    - delete: Link to delete the file using a DELETE request
+    - self: Link to the file's metadata
+    - updateMetadata: Link to update the file's metadata using a POST request
+    """
+
+    created: Optional[datetime] = Field(None, example="2018-05-15T18:54:27.519Z")
+    """
+    The date and time the file was created in the file service
+    """
+
+    id: Optional[str] = Field(None, example="5afb2ce3741fe11d88838cc9")
+    """
+    The file's ID within the service group
+    """
+
+    path: Optional[str] = Field(None, example="C:\\temp\\4afb2ce3741fe11d88838cc9.txt")
+    """
+    The path to the file on the server.  This field is returned only if
+    the user has associated privileges to view file paths.
+    """
+
+    properties: Optional[Dict[str, str]] = Field(None)
+    """
+    The file's properties
+    Example - {"Name": "myfile.txt", "MyProperty": "Value"}
+    """
+
+    meta_data_revision: Optional[int] = Field(None, alias="metaDataRevision", example=1)
```

Comment:
`metadata` isn't typically broken into two words

Reply:
It looks like the API implemented it as `metaDataRevision` and hence only `meta_data_revision` will result in a working automatic case conversion.

Reply:
It looks like the API implemented it as `metaDataRevision` and hence only `meta_data_revisio`n will result in a working automatic case conversion.